src/map/clif.c

```diff
@@ -9761,7 +9751,7 @@ static void clif_refresh(struct map_session_data *sd)
 	if (sd->charm_type != CHARM_TYPE_NONE && sd->charm_count > 0)
 		clif->charm_single(sd->fd, sd);
 	if (sd->soulball > 0)
-		clif->spiritball(&sd->bl, BALL_TYPE_SOUL, SELF);
+		clif->soulball(&sd->bl, sd->soulball, SELF);
```

Comment:
Like this one.

Reply:
oh, what I tried to say is that for `BALL_TYPE_SPIRIT`, `clif->spiritball` is already called with `AREA` every time, no need to "hardcode" it in the ternary (thus, simply using `target` is enough). Just trying to justify that removing the ternary was safe, because `clif->spiritball` was already called with AREA every time it was called for "spirit ball" instead of "soul ball".

This one was not `AREA`, but it was also not `BALL_TYPE_SPIRIT`